Add explanation for calendar (#14)
richardhj/contao-ajax_reload_element: README.md

</script>
```
### Ajax Calendar

> Add this to the `cal_default.html5` template and add a class like here `.inn_call_ajax` to the next and previous month links. In this example we have an article with the id `calendar` and the contao calendar as element.

Reviewer: ```suggestion
Add this to the `cal_default.html5` template and add the CSS classes `.calendar__next` and `.calendar__prev` to the next and previous month links. 
```

```html
<script>

>     $("#calendar a.inn_call_ajax").click(function (event) {

Reviewer: ```suggestion
    $(".calendar__next, .calendar__prev").click(function (event) {
```

        var element;
        //Get url of next Month

>         var $url = window.location.origin + '/' + $(this).attr('href');

Reviewer: I am not sure if an absolute link is required, but it is ok :)

Author: > Thank you very much! Can you please accept these suggestions? I changed the selector, so that the code is not dependent on a css id.

Welcome :) Thank you for your suggestions! :) have a nice day
